### ISSUE TITLE
About the training code of class "L_light"

### ISSUE THREAD

**Issue by psj-jerry** [cuiziteng/Illumination-Adaptive-Transformer#11]
Hi, I have seen the L_light class in the tools class introduced in the code for the enhancement task, but it is not used. After reading the code I feel that the L_light class is the Color Loss class in the Tools class. What does this class do in the code, please, and I'm rather curious how to use it? Thank you!

**Comment by cuiziteng** [comment]
Hello, do you mean the L_color in the util.py? This code is based on **gray-world assumption**, which I try to add the regularization on the A and M generated by local network, but it do not improve results, so I take off this loss function.

**Comment by psj-jerry** [comment]
Thank you! A great job~

**Comment by cuiziteng** [comment]
Thank you!
